descriptors/ModuleDescriptor-template.json

```diff
@@ -258,7 +258,9 @@
             "instance-authority-links.instances.collection.put",
             "instance-authority.linking-rules.collection.get",
             "user-tenants.collection.get",
-            "organizations.organizations.collection.get"
+            "organizations.organizations.collection.get",
+            "invoices.acquisitions-units-assignments.assign",
```

Comment:
we should not add this permission here, since this permission is optional and a user assigned to a particular acquisition unit should be directly provided with this permission. Otherwise, a user who is not supposed to be provided with this permission will receive it.

Reply:
Can it be added to permissionsDesired? we have it in mod-srm in that section? Or is it part of the permission set that needs to be assigned to the user?

Reply:
fixed